src/components/Calendar/Calendar.js

```diff
@@ -10,42 +10,56 @@ import { useNavigate } from "react-router";
 import GetAuthentication from "../Authentication/Authentification";
 import { PrimaryButton2 } from '../CustomMUIComponents/CustomButtons';
 import TripOriginIcon from '@mui/icons-material/TripOrigin';
-import IconButton from "@mui/material/IconButton";
-import MoreVertIcon from '@mui/icons-material/MoreVert';
-import MenuItem from '@mui/material/MenuItem';
 import axios from "axios";
 
 export default function CalendarView() {
 
-    const [date, setDate] = useState(new Date()) // stores date, sets date using Date obj
+    const [date, setDate] = useState(new Date()) 
 
-    const [event, setEvent] = useState([]);
+    const [events, setEvents] = useState([]);
+    const [acadEvents, setAcedemicEvents] = useState([]);
+
+    const [eventError, seteventError] = React.useState({ message: "Error, please try again later", hasError: false });
```

Comment:
What is the use of `eventError` state in this file? It is being set, but not used to display errors to the user (or anything else). Please remove this state if it's not being used.